src/main/java/jenkins/plugins/jobcacher/ArbitraryFileCache.java

```diff
@@ -454,6 +456,21 @@ public enum CompressionMethod {
                 GzipCompressorInputStream::new,
                 ".tgz")
         ),
+        TARGZ_BEST_SPEED(new TarArbitraryFileCacheStrategy(
+            os -> new GzipCompressorOutputStream(os, gzipParametersBestSpeed()),
+            GzipCompressorInputStream::new,
+            ".tgz")
+        ),
+        TARGZ_NO_COMPRESSION(new TarArbitraryFileCacheStrategy(
```

Comment:
Is there any actual benefit compared to "tar only"?

Reply:
Do you want me to remove the TARGZ_NO_COMPRESSION option? I guess there is no practical use for it, when one can use the TAR option.

Reply:
Yes, I think `TAR` is enough :)